### PR TITLE
Release 4.22 Updates

### PR DESCRIPTION
**Pull request by titusfortner**
### Description

* Version updates
* Release Blog Article
* Updated BiDi documentation & examples

### Status
Still need to:
1. Add image to blog
2. add enable bidi code to options examples
3. ~update translations~
4. ~put the low-level examples in Java and JS back~

### Feedback Needed

Everything under here: https://deploy-preview-1765--selenium-dev.netlify.app/documentation/webdriver/bidi/
Including all this: https://deploy-preview-1765--selenium-dev.netlify.app/documentation/webdriver/bidi/cdp/

### Motivation and Context
Selenium 4.22!

___

### **PR Type**
enhancement

___

### **Description**
- Updated Selenium package versions to 4.22.0 across multiple language examples including .NET, Java, JavaScript, Kotlin, Python, and Ruby.
- Ensured compatibility with the latest Selenium release.

___

### **Changes walkthrough** 📝
<table><thead><tr><th></th><th align="left">Relevant files</th></tr></thead><tbody><tr><td><strong>Dependencies
</strong></td><td><table>
<tr>
  <td>
    <details>
      <summary><strong>SeleniumDocs.csproj</strong><dd><code>Update .NET Selenium package versions to 4.22.0</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>
      
examples/dotnet/SeleniumDocs/SeleniumDocs.csproj

<li>Updated <code>Selenium.Support</code> package version to 4.22.0<br> <li> Updated <code>Selenium.WebDriver</code> package version to 4.22.0<br>

</details>
    

  </td>
  <td><a href="https://github.com/SeleniumHQ/seleniumhq.github.io/pull/1765/files#diff-71ac76d07f5edc5b438e23b53604457edcbd5d884bc745536306fb1356b741a1">+2/-2</a>&nbsp; &nbsp; &nbsp; </td>

</tr>                    

<tr>
  <td>
    <details>
      <summary><strong>pom.xml</strong><dd><code>Update Java Selenium version to 4.22.0</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>
      
examples/java/pom.xml

- Updated `selenium.version` property to 4.22.0

</details>
    

  </td>
  <td><a href="https://github.com/SeleniumHQ/seleniumhq.github.io/pull/1765/files#diff-57fef07b0713fb4af5de90aa728eeb03d0497763a3bc9ffe1d551f85b88271b4">+1/-1</a>&nbsp; &nbsp; &nbsp; </td>

</tr>                    

<tr>
  <td>
    <details>
      <summary><strong>package-lock.json</strong><dd><code>Update JavaScript Selenium package version in package-lock.json to </code><br><code>4.22.0</code></dd></summary>
<hr>
      
examples/javascript/package-lock.json

- Updated `selenium-webdriver` package version to 4.22.0

</details>
    

  </td>
  <td><a href="https://github.com/SeleniumHQ/seleniumhq.github.io/pull/1765/files#diff-9e9f1801be0cdd2ca5839a8a8bda05bc37424f1bc94c8f1b196282d65552e5bc">+7/-7</a>&nbsp; &nbsp; &nbsp; </td>

</tr>                    

<tr>
  <td>
    <details>
      <summary><strong>package.json</strong><dd><code>Update JavaScript Selenium package version in package.json to 4.22.0</code></dd></summary>
<hr>
      
examples/javascript/package.json

- Updated `selenium-webdriver` package version to 4.22.0

</details>
    

  </td>
  <td><a href="https://github.com/SeleniumHQ/seleniumhq.github.io/pull/1765/files#diff-e94247812c7153c9135cef3fd1883d9a76ab7d030651b66b943a675166366255">+1/-1</a>&nbsp; &nbsp; &nbsp; </td>

</tr>                    

<tr>
  <td>
    <details>
      <summary><strong>pom.xml</strong><dd><code>Update Kotlin Selenium version to 4.22.0</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>
      
examples/kotlin/pom.xml

- Updated `selenium.version` property to 4.22.0

</details>
    

  </td>
  <td><a href="https://github.com/SeleniumHQ/seleniumhq.github.io/pull/1765/files#diff-a31e7249367c27970f0bb46db6b84d134c93c9f931a48e0f187715904ca21ff8">+1/-1</a>&nbsp; &nbsp; &nbsp; </td>

</tr>                    

<tr>
  <td>
    <details>
      <summary><strong>requirements.txt</strong><dd><code>Update Python Selenium package version to 4.22.0</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>
      
examples/python/requirements.txt

- Updated `selenium` package version to 4.22.0

</details>
    

  </td>
  <td><a href="https://github.com/SeleniumHQ/seleniumhq.github.io/pull/1765/files#diff-c233c301c314607d2dfcee9a9ec0d5a7b78a67d2c8927808fd4c9d382bbb4a8e">+1/-1</a>&nbsp; &nbsp; &nbsp; </td>

</tr>                    

<tr>
  <td>
    <details>
      <summary><strong>Gemfile</strong><dd><code>Update Ruby Selenium gem version to 4.22.0</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>
      
examples/ruby/Gemfile

- Updated `selenium-webdriver` gem version to 4.22.0

</details>
    

  </td>
  <td><a href="https://github.com/SeleniumHQ/seleniumhq.github.io/pull/1765/files#diff-050c5bd1b060f5924d57c5aaeedad981932dd1db856e57ddf779dd5e34bd0d97">+1/-1</a>&nbsp; &nbsp; &nbsp; </td>

</tr>                    
</table></td></tr></tr></tbody></table>

___

> 💡 **PR-Agent usage**:
>Comment `/help` on the PR to get a list of all available PR-Agent tools and their descriptions

### CODE REVIEW

File: website_and_docs/content/blog/2024/selenium-4-22-released.md [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+  Today we're happy to announce that Selenium 4.22 has been released!
+---
+
+We're very happy to announce the release of Selenium 4.22.0 for 
+Javascript, Ruby, Python, .NET, Java and the Grid!
+Links to everything can be found on our [downloads page][downloads].
+
+### Highlights
+
+  * Selenium has at least [2.6 active users](https://plausible.io/manager.selenium.dev) in the last 30 days. 200k more than last month!
```
Comment (p0deje) [inline website_and_docs/content/blog/2024/selenium-4-22-released.md:18]: 2.6M!

Comment (titusfortner) [reply website_and_docs/content/blog/2024/selenium-4-22-released.md:18]: I know, right?

File: website_and_docs/content/documentation/webdriver/bidirectional/webdriver_bidi/browsing_context.ja.md
Comment (pujagani) [inline website_and_docs/content/documentation/webdriver/bidirectional/webdriver_bidi/browsing_context.ja.md:1]: Folks interested in BiDi (contributors and users) often ask for documentation and it is easy to point them to this section to start with. Our messaging is that Selenium supports projects that build opinionated APIs using Selenium APIs so we need documentation and code examples for both high-level API and low-level APIs.

**Review by pujagani — CHANGES_REQUESTED** [review]

File: website_and_docs/content/blog/2024/selenium-4-22-released.md [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+    * Python, Chrome and Windows all see the majority of use.
+* Chrome DevTools support is now: v124, v125, and v126 (Firefox still uses v85 for all versions)
+* The first implementations of the new [BiDi API](https://www.selenium.dev/documentation/webdriver/bidi/logging)
+have rolled out in Ruby, Python and JavaScript
+
+#### Noteworthy changes per language
+
+  * Java 
+    * Enabling BiDi can now be accomplished by calling `enableBiDi()` on an Options class instance.
+    * Video file name in Dynamic Grid can be seet with `se:videoName` capability.
```
Comment (pujagani) [inline website_and_docs/content/blog/2024/selenium-4-22-released.md:30]: nit "set"

**Pull request merged**